mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -719,7 +740,10 @@ func expandProviderSetting(d *schema.ResourceData) matlas.ProviderSettings {
 	providerSettings := matlas.ProviderSettings{}
 
 	if d.Get("provider_name") == "AWS" {
-		providerSettings.DiskIOPS = pointy.Int64(cast.ToInt64(d.Get("provider_disk_iops")))
+		// Asks if the Provider Disk IOS was set if it didn't the server will set it up automatically
```

Comment:
Just checking but you are noting in the comment here that if it's not included in the terraform config Atlas just sets it to the default for the amount of storage, correct?

Reply:
yes, it's correct

Reply:
Great, perhaps clarify in the comment for others?

Reply:
Yes, it's a great idea thanks